### PR TITLE
(MAINT) Release prep for PDK 2.1.0.0

### PR DESCRIPTION
**Pull request by sanfrancrisko**

### CODE REVIEW

File: docs/release_notes_pdk.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+
+### Resolved issues
+
+#### `--verbose` option broken for  `pdk new defined_type`
+
+With [ddfreyne/cri#103](https://github.com/ddfreyne/cri/issues/103) merged, this issue has been resolved.
+
+#### PDK output scrollback in VS Code limited
+
+By disabling ansicon on Windows when running PDK in the integrated terminal of VS Code, this issue is now resolved.
```
Comment (jpogran) [inline docs/release_notes_pdk.md:21]:
```suggestion
Resolved an issue using PDK in the integrated terminal of VS Code by disabling ansicon on Windows .
```

File: docs/release_notes_pdk.md [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
 
 New features, enhancements, and resolved issues for the PDK 2.x release series.
+
+## PDK 2.1
+
+### New features and enhancements
+
+#### Add `pdk env` subcommand
+
+Adds an experimental `pdk env` subcommand, which aids in setting a CLI context for a specified version of Puppet (https://github.com/nkanderson).
```
Comment (jpogran) [inline docs/release_notes_pdk.md:11]:
```suggestion
Adds an experimental `pdk env` subcommand, which prints the current environment variables used by the PDK (https://github.com/nkanderson).
```

**Review by jpogran — CHANGES_REQUESTED** [review]

**Review by da-ar — APPROVED** [review]

**Comment by hestonhoffman** [comment]
@sanfrancrisko Could you bump the version number here: https://github.com/sanfrancrisko/pdk/blob/maint/main/2_1_0_0_release_prep_docs/docs/pdk.md

File: docs/release_notes_pdk.md [hunk 3]
```diff
@@ -1,6 +1,31 @@
...
+
+#### Add `pdk env` subcommand
+
+Adds an experimental `pdk env` subcommand, which prints the current environment variables used by the PDK (https://github.com/nkanderson).
+
+### Resolved issues
+
+#### `--verbose` option broken for  `pdk new defined_type`
+
+With [ddfreyne/cri#103](https://github.com/ddfreyne/cri/issues/103) merged, this issue has been resolved.
```
Comment (hestonhoffman) [inline docs/release_notes_pdk.md:17]:
```suggestion
Fixes an issue where the `--verbose` option would not work with the `pdk new defined_type` command. See [ddfreyne/cri#103](https://github.com/ddfreyne/cri/issues/103).
```

File: docs/release_notes_pdk.md [hunk 4]
```diff
@@ -1,6 +1,31 @@
...
 
 New features, enhancements, and resolved issues for the PDK 2.x release series.
+
+## PDK 2.1
+
+### New features and enhancements
+
+#### Add `pdk env` subcommand
+
+Adds an experimental `pdk env` subcommand, which prints the current environment variables used by the PDK (https://github.com/nkanderson).
```
Comment (hestonhoffman) [inline docs/release_notes_pdk.md:11]:
```suggestion
Adds an experimental `pdk env` subcommand. Use `pdk env` to print the environment variables that PDK is currently using (https://github.com/nkanderson).
```

File: docs/release_notes_pdk.md [hunk 5]
```diff
@@ -1,6 +1,31 @@
...
+
+#### `--verbose` option broken for  `pdk new defined_type`
+
+With [ddfreyne/cri#103](https://github.com/ddfreyne/cri/issues/103) merged, this issue has been resolved.
+
+#### PDK output scrollback in VS Code limited
+
+Resolved an issue using PDK in the integrated terminal of VS Code by disabling ansicon on Windows.
+
+### Template Changes and Fixes
```
Comment (hestonhoffman) [inline docs/release_notes_pdk.md:23]:
```suggestion
### Template changes and fixes
```

File: docs/release_notes_pdk.md [hunk 6]
```diff
@@ -1,6 +1,31 @@
...
+
+* Relocation of `inventory.yaml` file used by Litmus. This prevents accidental execution of acceptance tests in production systems. (https://github.com/pmcmaw)
```
Comment (hestonhoffman) [inline docs/release_notes_pdk.md:25]:
```suggestion
* Relocated the`inventory.yaml` file used by Litmus. This prevents accidental execution of acceptance tests in production systems. (https://github.com/pmcmaw)
```

File: docs/release_notes_pdk.md [hunk 7]
```diff
@@ -1,6 +1,31 @@
...
+* Removal of win32 gems from Gemfile. These have been included in Puppet since version 5. (https://github.com/da-ar)
+* Add Puppet 7 tests to .gitlab-ci.yml (https://github.com/silug)
```
Comment (hestonhoffman) [inline docs/release_notes_pdk.md:27]:
```suggestion
* Added Puppet 7 tests to .gitlab-ci.yml (https://github.com/silug)
```

File: docs/release_notes_pdk.md (hunk 6 above)
Comment (hestonhoffman) [reply docs/release_notes_pdk.md:25]: These template bullets have historically been in the past tense. I think that works better for these.

File: docs/release_notes_pdk.md (hunk 5 above)
Comment (hestonhoffman) [reply docs/release_notes_pdk.md:23]: If you could fix this on line 75 as well, that would be great :)

**Pull request merged**